components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java

```diff
@@ -674,6 +668,36 @@ private void addOrUpdateOIDCSpProperty(OAuthAppDO oauthAppDO,
         prepStatementForPropertyDelete.executeBatch();
     }
 
+    private void addAudiencestoSpProperty(String preprocessedClientId, int spTenantId, String[] audiences,
```

Comment:
shall we rename the method as `updateAudiences`?

Reply:
fixed wit d43a24390e23624cc0139558087860229e9dbe7e